### DISCUSSION TITLE
Welcome to GitLab2PROV Discussions!

### DISCUSSION THREAD

**Discussion by onyame** [DLR-SC/gitlab2prov#50] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by LuigiaCristiano** [comment]
Hi, I do have a question on the usage of the GitLab2PROV.
I make just a generic usage example.
I do have a gitlab repo where I did commits also for small typo correction.
I want to have in my PROV file only "meaningful" commits.
This can be achieved by a proper labeling of the commits.

Can I do explicitate which actions I want to collect in the PROV file at creation time or do I need to select the commits that are relevant for me afterwords?
Do you have a tagging or labeling strategy of gitlab commits that you would suggest to users of your GitLab2PROV package ?
Thanks
Luigia
